granulate_utils/containers/docker.py

```diff
@@ -36,13 +36,13 @@ def __init__(self) -> None:
         self._docker = docker.DockerClient(base_url="unix://" + ns.resolve_host_root_links(DOCKER_SOCK))
 
     def list_containers(self, all_info: bool, only_running: bool = True) -> List[Container]:
-        container_filter = None
+        container_filters = None
```

Comment:
From the previous PR actually, but is it None by default? Did you verify if the function accepts None vs empty dict?